web/packages/teleterm/src/ui/services/workspacesService/documentsService/types.ts

```diff
@@ -227,30 +227,26 @@ export interface DocumentVnetInfo extends DocumentBase {
   // the document fields, hence why rootClusterUri is defined here.
   rootClusterUri: uri.RootClusterUri;
   /**
-   * Details of the app if the doc was opened by selecting a specific TCP app.
+   * Details of the resource if the doc was opened by selecting a specific resource.
```

Comment:
I think it might make sense to move this JSDoc comment to the definition of `VnetLauncherArgs` rather than keeping it above the `launcherArgs` field.